src/geowombat/core/geoxarray.py

```diff
@@ -1540,21 +1536,20 @@ def moving(
             >>>
             >>> # Calculate the mean within a 5x5 window
             >>> with gw.open('image.tif') as src:
-            >>>     res = src.gw.moving(stat='mean', w=5, nodata=32767.0, n_jobs=8)
+            >>>     res = src.gw.moving(stat='mean', w=5, nodata=32767.0)
             >>>
             >>> # Calculate the 90th percentile within a 15x15 window
             >>> with gw.open('image.tif') as src:
-            >>>     res = src.gw.moving(stat='perc', w=15, perc=90, nodata=32767.0, n_jobs=8)
+            >>>     res = src.gw.moving(stat='perc', w=15, perc=90, nodata=32767.0)
+            >>>     res.data.compute(num_workers=4)

```

Comment:
Looks like this might affect other functions using `num_workers`.  Should I scan the documentation?

Reply:
I didn't find any other occurrences, but feel free to take a look.